techmeet/apps/frontend/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM node:10-alpine
+MAINTAINER Bryan Hyshka <bryan@hyshka.com>
+
+# Prefix path with global node_modules folder
+# This allows npm package binaries to be available everywhere
+ENV PATH /usr/src/app/node_modules/.bin:$PATH
+
+# Set variable that we use in our webpack config to detect
+# that we're running the stack inside of Docker
+ENV VIRTUAL_ENV docker
+
+# Update, install, and cleanup
+# RUN apk update \
+    # && apk add git \
```

Comment:
Best to leave this off if it's commented out.

Reply:
Removed in https://github.com/keeners/techmeet/pull/11/commits/09ab9c85f12571b54d5ec4f790afe24f3ed1ff60